tests/test_adapters.py

```diff
@@ -43,6 +43,7 @@
     record_mode='none',
     match_on=['uri', 'method'],
     filter_headers=['x-auth-token'],
+    filter_query_parameters=['timeout'],
```

Comment:
No especially familiar with VCR so don't really understand what's going on here, nor in the `triangle-ising.yaml`.

Reply:
Need to ignore `timeout` query parameter (or refresh all recorded responses), because cloud-client 0.13 sets it for long-polling requests.

Reply:
VCR.py is nice, I'd use it for cloud-client, if I were to do them from scratch.